### ISSUE TITLE
Artillery runs forever and never exit

### ISSUE THREAD

**Issue by endyjasmi** [shoreditch-ops/artillery#237]
Whenever the command `artillery run config.json` is executed, the artillery will run fine for the first few request and later keep on looping with the following progress report and never end.
```bash
Report for the previous 10s @ 2017-01-06T04:08:56.069Z
  Scenarios launched:  0
  Scenarios completed: 0
  Requests completed:  0
  RPS sent: NaN
  Request latency:
    min: NaN
    max: NaN
    median: NaN
    p95: NaN
    p99: NaN
  Scenario duration:
    min: NaN
    max: NaN
    median: NaN
    p95: NaN
    p99: NaN
 -  artillery:runner max worker concurrency: {} +10s
```

Following is the environment it runs in:
```bash
root@nodejs-512mb-ams3-01:~/artillery# node -v
v6.9.2
root@nodejs-512mb-ams3-01:~/artillery# artillery -V
1.5.0-21
```

Following is the config file:
```json
{
	"config": {
		"target": "https://www.company.com",
		"phases": [
			{ "duration": 60, "arrivalRate": 10 }
		]
	},
	"scenarios": [
		{
			"flow": [
				{"get": { "url": "/" }},

				{"get": { "url": "/js/vendor/picturefill.min.js" }},
				{"get": { "url": "/js/vendor/modernizr-2.6.2-respond-1.1.0.min.js" }},
				{"get": { "url": "/js/vendor/es5-shim.min.js" }},
				{"get": { "url": "/js/vendor/es6-shim.min.js" }},
				{"get": { "url": "/js-localization/localization.js" }},
				{"get": { "url": "/js-localization/messages" }},
				{"get": { "url": "/js/vendor/require.min.js" }},
				{"get": { "url": "/dist/app.js?bust=20171481838" }},

				{"get": { "url": "/css/mainv2.css?bust=2016126105156" }},

				{"get": { "url": "/img/company_logo/company_logo_full_width320x66.png" }},
				{"get": { "url": "/img/index/more_section_artisans_desktop.jpg" }},
				{"get": { "url": "/img/index/more_section_grower_desktop.jpg" }},
				{"get": { "url": "/img/index/more_section_customer2_desktop.jpg" }},
				{"get": { "url": "/img/index/sizes_section_desktop.jpg" }},
				{"get": { "url": "/img/index/sizes_section_mobile.jpg" }},
				{"get": { "url": "/img/index/fresh_section_desktop.jpg" }},
				{"get": { "url": "/img/index/fresh_section_mobile.jpg" }},
				{"get": { "url": "/img/index/designs_section2_desktop.jpg" }},
				{"get": { "url": "/img/index/designs_section1_desktop.jpg" }},
				{"get": { "url": "/img/index/happiness_section_desktop.jpg" }},
				{"get": { "url": "/img/howitworks/step3_icon.png" }},
				{"get": { "url": "/img/howitworks/step2_icon.png" }},
				{"get": { "url": "/img/howitworks/step1_icon.png" }},
				{"get": { "url": "/img/index/how_it_works_section_desktop.jpg" }},
				{"get": { "url": "/img/index/how_it_works_section_mobile.jpg" }},
				{"get": { "url": "/img/company_logo/seal_desktop80x80.png" }},
				{"get": { "url": "/video/ourprocess/video_our_process.jpg" }},
				{"get": { "url": "/img/index/happiness_section_mobile.jpg" }},
				{"get": { "url": "/img/company_logo/company_logo_desktop195x40.png" }},
				{"get": { "url": "/img/company_logo/company_logo_mobile145x30.png" }},

				{"get": { "url": "/video/ourprocess/video_our_process.mp4" }},

				{"get": { "url": "/fonts/trade_gothic_std_bold2.woff" }},
				{"get": { "url": "/fonts/trade_gothic.woff" }},

				{"get": { "url": "/favicons/favicon-96x96.png" }},
				{"get": { "url": "/favicons/favicon-16x16.png" }}
			]
		}
	]
}
```

Following is the log file generated by running `DEBUG=* artillery run config.json 2>&1 | tee debug.log`:
https://gist.github.com/endyjasmi/16f1a7951574fc0efd8a9795b7b5c16e

Thanks in advance for the helps.

**Comment by hassy** [comment]
thanks for the bug report @endyjasmi

**Comment by hassy** [comment]
@endyjasmi should be fixed in `1.5.0-22`
